setup.py

```diff
@@ -524,15 +532,15 @@ def get_sphinx_theme_version() -> str:
 
 # Dict of all providers which are part of the Apache Airflow repository together with their requirements
 PROVIDERS_REQUIREMENTS: Dict[str, List[str]] = {
-    'airbyte': [],
+    'airbyte': http,
     'amazon': amazon,
     'apache.beam': apache_beam,
     'apache.cassandra': cassandra,
     'apache.druid': druid,
     'apache.hdfs': hdfs,
     'apache.hive': hive,
     'apache.kylin': kylin,
-    'apache.livy': [],
+    'apache.livy': http,
```

Comment:
```suggestion
    'apache.livy': [],
```

Already depends on http provider